src/actions/channels.js

```diff
@@ -118,7 +118,7 @@ export function createDirectChannel(userId: string, otherUserId: string): Action
             {user_id: userId, category: Preferences.CATEGORY_CHANNEL_OPEN_TIME, name: created.id, value: new Date().getTime().toString()},
         ];
 
-        savePreferences(userId, preferences)(dispatch);
+        savePreferences(userId, preferences)(dispatch, getState);
```

Comment:
This should be `dispatch(savePreferences(userId, preferences));` the same applies to the other places as well.

Reply:
All changes in this file I did only to pass flow check, ok I'll change them to dispatch actions